src/components/TrainStats.tsx

```diff
@@ -0,0 +1,34 @@
+import * as stats from '../../static/static_data.json';
+import '../styles/TrainStats.css'
+
+interface IStats {
+    line: string;
+}
+
+export const Stats = ({line}: IStats): JSX.Element => {
+    return (
+        <details className='stats-container'>
+            <summary className='stats-title'>Stats for {line} line</summary>
+            <table className='stats-table'>
+                <tbody>
+                    <tr>
+                        <td>New Trains Delivered:</td>
+                        <td>{stats[line]?.totalNewDelivered}</td>
```

Comment:
Might be a little cleaner to extract `stats[line]` into a variable and `return null` early from this component if it doesn't exist for some reason? Otherwise the fallback behavior is to show these controls but have them contain no info.